Python/Product/Core/Properties/AssemblyInfo.cs

```diff
@@ -41,3 +41,6 @@
 [assembly: ProvideCodeBase(AssemblyName = "Microsoft.PythonTools.VSCommon", CodeBase = "Microsoft.PythonTools.VSCommon.dll", Version = AssemblyVersionInfo.StableVersion)]
 [assembly: ProvideCodeBase(AssemblyName = "Microsoft.PythonTools.VSInterpreters", CodeBase = "Microsoft.PythonTools.VSInterpreters.dll", Version = AssemblyVersionInfo.StableVersion)]
 [assembly: ProvideCodeBase(AssemblyName = "Microsoft.PythonTools.Workspace", CodeBase = "Microsoft.PythonTools.Workspace.dll", Version = AssemblyVersionInfo.StableVersion)]
+
+// HACK: Helps running on 15.5
+[assembly: ProvideCodeBase(AssemblyName = "Microsoft.VisualStudio.Debugger.Interop.VSCodeDebuggerHost", CodeBase = "Microsoft.VisualStudio.Debugger.Interop.VSCodeDebuggerHost.dll")]
```

Comment:
Probably don't want to merge that, right?

Reply:
Correct, especially since it isn't even necessary with the other fix I put in.